nuxeo-coldstorage-web/.eslintrc.json

```diff
@@ -0,0 +1,44 @@
+{
+  "extends": ["airbnb-base", "prettier"],
```

Comment:
Is there any reason to use a different ruleset than the one used in Web UI?

Reply:
Yes, I guess there are some... to start, Web UI and Elements use different rule sets. 😅 
This kind of set is closer to what we have on Elements (and we've already being using this on Aspera).

Ideally, we could factorize this kind of stuff so we can have consistent styling (and maybe unit test configs, etc.).
There is a ticket lost on our backlogs to start this kind of move (ELEMENTS-1166)